shared/components/CameraView.js

```diff
@@ -108,6 +108,20 @@ class CameraView extends Component {
     };
   }
 
+  componentDidMount() {
+    navigator.geolocation.getCurrentPosition(
```

Comment:
I think, this function can be moved in map view. becuase, we have to determine what is the first location in map first.